packages/editor/src/store/selectors.js

```diff
@@ -1487,6 +1487,17 @@ export function isAutosavingPost( state ) {
 	return isSavingPost( state ) && state.saving.isAutosave;
 }
 
+/**
+ * Returns true if the post is being previewed, or false otherwise.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean} Whether the post is being previewed.
+ */
+export function isPreviewingPost( state ) {
```

Comment:
It's slightly weird that preview would have this side-effect but it wouldn't be evident from selector name. Though I guess it matches the expectation of the action itself.